operations/mimir-mixin-compiled-baremetal/dashboards/mimir-tenants.json

```diff
@@ -2442,7 +2442,7 @@
                   "steppedLine": false,
                   "targets": [
                      {
-                        "expr": "sum(cortex_query_scheduler_queue_length{cluster=~\"$cluster\", job=~\"($namespace)/((ruler-query-frontend.*))\", user=\"$user\"})",
+                        "expr": "sum(cortex_query_scheduler_queue_length{cluster=~\"$cluster\", job=~\"($namespace)/((ruler-query-scheduler.*))\", user=\"$user\"})",
```

Comment:
addressed in https://github.com/grafana/mimir/pull/4152/commits/6b8f5985572f04df64e865c69712b72f88d2e980

Reply:
> Because the query-scheduler is an optional component, the queue is contained in the query-frontend if the scheduler isn't running. So I think we want to use either one of these components for this metric.

No, the query-frontend exposes a different metric: `cortex_query_frontend_queue_length`